third_party/compilerworks/cw_stdlib.sql

```diff
@@ -0,0 +1,495 @@
+/*
```

Comment:
I don't see any tests for these UDFs... 

@danieldeleo how hard would it be to run the UDF tests also in third_party?

Reply:
Removed UDFs